index.html

```diff
@@ -9,11 +9,11 @@
   <body>
     <div>
       <header>
-        <h2 class="studentName">Student Page</h2>
+        <h2 class="studentName">Hamzah's Page</h2>
       </header>
 
       <section class="container">
-        <h3 class="aboutMe">About me</h3>
+        <h3 class="aboutMe">Trying to learn something new.</h3>
```

Comment:
Well Done! :star2: 
Just change this header so it has the **About me** title, and add below it your description in a `<p>` tag not on the same `<h3>`
like so:
```html
        <h3 class="aboutMe">About me</h3>
        <p> your Description here </p>
```